content/windows.md

```diff
@@ -252,6 +115,14 @@ It may take several minutes or more for the process to finish -- wait until the
 
 ---
 
+## Dual-Booting Windows with Linux
+
+Dual booting allows you to run 2 perating systems on the same computer. There are several ways to dual boot Pop!_OS and Windows. Make sure to follow the instructions for your specific situation:
```

Comment:
We would typically write out numbers under 10: https://www.grammarly.com/blog/when-to-spell-out-numbers/

"operating" is missing an "o"